pkg/asset-manager-utils/test/RewardsAssetManager.test.ts

```diff
@@ -413,22 +451,16 @@ describe('Rewards Asset manager', function () {
       });
 
       context('when fee percentage is non-zero', () => {
-        let targetInvestmentAmount: BigNumber;
-        const poolConfig = {
-          targetPercentage: fp(0.5),
-          upperCriticalPercentage: fp(1),
-          lowerCriticalPercentage: fp(0.1),
-          feePercentage: fp(0.1),
-        };
         sharedBeforeEach(async () => {
-          poolController = lp; // TODO
+          const poolController = lp; // TODO
 
           await assetManager.connect(poolController).setPoolConfig(poolId, poolConfig);
-          targetInvestmentAmount = await assetManager.maxInvestableBalance(poolId);
         });
 
         it('returns the expected fee', async () => {
```

Comment:
Not sure what sort of refactor you have in mind, but for these I'd consider not having a full `describe` just for `getRebalanceFee` (with their complex setup and difference scenarios), but instead add these as an `it` in the multiple contexts of `describe('rebalance')`, e.g. `it('returns the expected fee'); it('emits an event');`, etc.